article.html

```diff
@@ -0,0 +1,683 @@
+                              <img src="./assets/yt.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="YouTube">
+                            </div>
+                          </div>
+
+                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg" data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg" data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                              <img src="./assets/linkedin.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="LinkedIn">
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid article aem-GridColumn aem-GridColumn--default--12 layout-grid">
+            <div id="article-container" class="cmp-container">
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                    <div class="text date quote-content">
+                        <div id="text-49e22e7256" class="cmp-text">
+                            <p>November 28, 2022 </p>
+                        </div>
+                    </div>
+                    <div class="container responsivegrid article-top aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div id="article-top-container" class="cmp-container">
+                            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                                <div class="container responsivegrid heading-author aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div id="container-445633ddcf" class="cmp-container">
+                                        <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                                            <div class="title heading aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <div id="title-63091b8133" class="cmp-title">
+                                                  <h1 class="cmp-title__text">Uncommon </br> Opportunities to Lead</h1>
```

Comment:
Please style this to create the word wrap without the `<br />`. This will work better for responsiveness.